spec/spec_helper_acceptance.rb

```diff
@@ -48,11 +50,11 @@ def run_device(options = { allow_changes: true })
 end
 
 def run_resource(resource_type, resource_title = nil)
-  if resource_title
-    result = Open3.capture2e("bundle exec puppet device --target sut --resource #{resource_type} #{resource_title} --trace --modulepath spec/fixtures/modules --deviceconfig spec/fixtures/acceptance-device.conf")
-  else
-    result = Open3.capture2e("bundle exec puppet device --target sut --resource #{resource_type} --trace --modulepath spec/fixtures/modules --deviceconfig spec/fixtures/acceptance-device.conf")
-  end
+  result = if resource_title
+             Open3.capture2e("bundle exec puppet device --resource #{resource_type} #{resource_title} #{COMMON_ARGS} --verbose --trace")
```

Comment:
For the acceptance tests I would expect `--verbose --trace --debug` to be part of the `COMMON_ARGS`

Reply:
Can we have `--debug` in the `puppet device` calls? Should give better details if there are failures on CI